recipe/meta.yaml

```
@@ -117,22 +121,24 @@ requirements:
     - nspr                               # [unix]
     - nss                                # [unix]
     - libsqlite
-    - mysql-devel                        # [not win]
+    - mysql-devel 8.0                    # [not win]
```

```suggestion
    - mysql-devel 8.0                    # [not win]
```
This will completely undo the effect of the migrator.

---

Thank you for finding this. i guess i was too excited!